source/fundamentals/enterprise-auth.txt

```diff
@@ -1,27 +1,186 @@
+.. _golang-enterprise-authentication-mechanisms:
+
 ====================================
 Enterprise Authentication Mechanisms
 ====================================
 
-.. default-domain:: mongodb
-
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
 
-In this guide, we show you how to authenticate with MongoDB using each
-**authentication mechanism** available exclusively in the MongoDB Enterprise
-Edition.
+Overview
+--------
+
+In this guide, you can learn how to authenticate with MongoDB using the
+**authentication mechanisms** available in the MongoDB Enterprise Edition.
+Authentication mechanisms are processes by which the driver and server confirm
+the identity of a client to ensure security before connecting.
 
-You can use the following mechanisms with the latest version of the MongoDB
-Enterprise Edition:
+You can use the following authentication mechanisms with the latest version of
+the MongoDB Enterprise Edition.
 
-Kerberos (GSSAPI)
-LDAP (PLAIN)
+- :ref:`GSSAPI/Kerberos <golang-kerberos>`
+- :ref:`LDAP (Plain) <golang-LDAP>`
 
-To authenticate using another mechanism, see
-:ref:`golang-authentication-mechanisms`. To learn more about 
-establishing a connection to your MongoDB cluster, see 
+To authenticate using another mechanism, see the
+:ref:`<golang-authentication-mechanisms>` fundamentals page. For
+more information on establishing a connection to your MongoDB cluster, see the
 :ref:`golang-connection-guide`.
 
+You can specify your authentication mechanism and credentials when connecting to
+MongoDB using the following methods:
+
+- Using a **connection string**, which is a string
+  that tells the driver how to connect to a MongoDB deployment and how to behave while
+  connected.
+
+- Specifying credentials and an authentication mechanism in the
+  ``Credential`` type from the {+driver-short+}. You can then use your
+  ``Credential`` struct to provide options for the ``Client``.
+
+This guide provides information on using types and methods provided by
+the {+driver-short+} for enterprise authentication, rather than using a
+connection string. For more information on using a connection string for
+enterprise authentication, see the :manual:`manual entry on Connection
+String URIs </reference/connection-string/#authentication-options>`.
+
+.. _golang-kerberos:
+
+Authenticate with GSSAPI/Kerberos
+---------------------------------
+
+The Generic Security Services API (GSSAPI) authentication mechanism allows the 
+user to authenticate to a Kerberos service using the user's principal name.
+
+The following example specifies the authentication mechanism using the following
```

Comment:
Issue: the example is introduced twice, so it was unclear where it begins. It's also unclear to me where it ends because rather than an example, I see placeholders and specifying the Credential struct.

Suggestion:
I think restructuring this section using the following ordering could be helpful:
1) What is this mechanism (already your first paragraph)
2) What are the requirements (build tag, compilation support, specifying credentials in Credential struct, alternatively keytab, etc)
3) Example

Reply:
I added a heading to delineate the example from the introductory text for each mechanism

Reply:
The heading looks good. However, I think the restructuring is still necessary -- I left another related comment [here](https://github.com/mongodb/docs-golang/pull/186/files#r1016913399) with an additional explanation.